cmd/validate_image.go

```diff
@@ -98,14 +99,22 @@ Return a non-zero status code on validation failure:
 
   ec validate image --image registry/name:tag --strict
 
+Return a summary of test results:
+
+  ec validate image --image registry/name:tag --policy my-policy --summary
+
 Use an EnterpriseContractPolicy resource from the currently active kubernetes context:
 
   ec validate image --image registry/name:tag --policy my-policy
 
 Use an EnterpriseContractPolicy resource from a different namespace:
 
   ec validate image --image registry/name:tag --policy my-namespace/my-policy
+  
+Use an EnterpriseContractPolicy resource from a different namespace:
```

Comment:
This example looks identical to the one above (including the description)

Reply:
yep, copy/paste mistake.